orchagent/mplsrouteorch.cpp

```diff
@@ -142,74 +146,92 @@ void RouteOrch::doLabelTask(Consumer& consumer)
                     if (fvField(i) == "mpls_pop")
                         pop_count = to_uint<uint8_t>(fvValue(i));
 
-                    if (fvField(i) == "blackhole")
-                        blackhole = fvValue(i) == "true";
+                    if (fvField(i) == "nexthop_group")
```

Comment:
Please restore the blackhole support for MPLS routes.

Reply:
Fixed